src/components/ViewPCOL/NotesForm.tsx

```diff
@@ -112,32 +108,30 @@ const NotesForm = ({
       <br />
```

Comment:
This is for line 106 (maxLength) -- This enforces I can't type past 2k characters, but there is nothing in the UI to tell me what that limit is, or that I've reached it, other than I can't type anymore. When pasting, it just truncates, which may or may not be noticed. Not a huge deal as they shouldn't have notes this long..